cmake/cuda.cmake

```diff
@@ -200,9 +200,9 @@ elseif(CMAKE_BUILD_TYPE  STREQUAL "MinSizeRel")
 endif()
 else(NOT WIN32)
 if(CMAKE_BUILD_TYPE  STREQUAL "Debug")
-    list(APPEND CUDA_NVCC_FLAGS  "-g -G")
+    list(APPEND CUDA_NVCC_FLAGS  "-g -G --compiler-options;/bigobj")
```

Comment:
can we put the bigobj in one place? I remember that we have define them in parent scope.